backend/src/main/java/com/github/lenaruppert/backend/controller/MongoUserController.java

```diff
@@ -0,0 +1,102 @@
+package com.github.lenaruppert.backend.controller;
+
+import com.github.lenaruppert.backend.model.MongoUser;
+import com.github.lenaruppert.backend.model.MongoUserDTO;
+import com.github.lenaruppert.backend.repository.MongoUserRepository;
+import jakarta.servlet.http.HttpSession;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.server.ResponseStatusException;
+
+import java.security.Principal;
+import java.util.UUID;
+
+@RestController
+@RequestMapping("api/user")
+@RequiredArgsConstructor
+public class MongoUserController {
+
+    private final MongoUserRepository mongoUserRepository;
+    private final PasswordEncoder passwordEncoder;
+
+    @PostMapping
+    public MongoUser create(@RequestBody MongoUserDTO user) {
+        if (user.username() == null || user.username().length() == 0) {
```

Comment:
```suggestion
        if (user.username() == null || user.username().isBlank()) {
```
With `isBlank` you also check for a string containing only white spaces :)